deploy/charts/rook-ceph/values.yaml

```diff
@@ -347,17 +347,17 @@ csi:
   #rbdLivenessMetricsPort: 9080
   #kubeletDirPath: /var/lib/kubelet
   #cephcsi:
-    #image: quay.io/cephcsi/cephcsi:v3.6.1
```

Comment:
These are intended to be commented out so that the defaults in the code will be picked up. But we need to revert these changes for readability since if someone wants to override the images, they need to preserve that indentation.

Reply:
reverted